netbox/virtualization/models.py

```diff
@@ -267,9 +267,10 @@ class VirtualMachine(ChangeLoggedModel, ConfigContextModel, CustomFieldModel):
     ]
 
     STATUS_CLASS_MAP = {
-        'active': 'success',
-        'offline': 'warning',
-        'staged': 'primary',
+        VirtualMachineStatusChoices.STATUS_ACTIVE: 'success',
+        VirtualMachineStatusChoices.STATUS_OFFLINE: 'warning',
+        VirtualMachineStatusChoices.STATUS_STAGED: 'primary',
+        VirtualMachineStatusChoices.STATUS_DECOMMISSIONING: 'warning',
```

Comment:
We should use different classes for the offline and decommissioning statuses. Might want to change offline to `danger` (red).

Reply:
That is better. Changed the color of "OFFLINE" to `danger`

Reply:
color label of vm status is set the same color as that of device.